templates/treat_ad.yaml

```diff
@@ -0,0 +1,61 @@
+Structural_Biology_Core_Folder: &struct_bio_folder
+  name: Structural_Biology_Core
+  type: Folder
+  next_level:
```

Comment:
It doesn't have to have this name, but I thought it was a fairly simple way of keeping the tree structure clean! 🤷🏻‍♀️

Reply:
In my example, I was hoping to avoid the extra nesting, but this is simple and clean. 

As for the name, I think `children` might be more consistent with existing Synapse APIs/clients.

Reply:
I love the name `children` for this.  Much better.